release/api/v1alpha1/bundle_types.go

```diff
@@ -241,6 +241,7 @@ type PackageBundle struct {
 	Controller     Image  `json:"packageController"`
 	TokenRefresher Image  `json:"tokenRefresher"`
 	HelmChart      Image  `json:"helmChart,omitempty"`
+	Bundle         Image  `json:"bundle,omitempty"`
```

Comment:
Any reason we want to add the bundle to the release? I thought the whole goal was to remove dependency on EKSA CLI release.

Reply:
The goal is to remove dependency of packages from the CLI, not CLI from packages. The EKS-A bundle should contain reference to everything it installs. The package bundle is currently hard coded in the CLI rather than extracted from EKS-A bundle